packages/entando-apimanager/modules/index.js

```diff
@@ -5,6 +5,7 @@ export {
   config,
   METHODS,
 } from './api/apiManager';
+export { default as APIProvider } from './api/APIProvider';
```

Comment:
this is a component, all the api stuff is about just business logic to perform http requests.
move it to a new directory named `components` directly inside `modules`

Reply:
ugh sorry, I wasn't aware I place it inside api folder. my bad

Reply:
done